lib/tasks_private/regions.rake

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+namespace 'aws:extract' do
+  desc 'Get / renew regions list from online source of AWS gem'
+  task :regions do
+    require 'uri'
+    require 'json'
+    require 'yaml'
+    require 'net/http'
+
+    # path valid for aws-sdk v3 gem's source
+    uri = URI('https://raw.githubusercontent.com/aws/aws-sdk-ruby/master/gems/aws-partitions/partitions.json')
+    out_file = ManageIQ::Providers::Amazon::Engine.root / 'lib' / 'data' / 'aws_regions.yaml'
```

Comment:
Same on the `root.join`.  However, you may want to make the location a constant in the ManageIQ::Providers::Amazon::Engine class, so that you don't have the location duplicated in multiple places.

Reply:
I prefer to avoid full environment loading for a such simple task.

Reply:
I don't understand what that means considering you still need to access the same constant either way.

Reply:
@Fryguy will it be better if I'll use something like `ManageIQ::Providers::Amazon::Engine::Paths::Fixtures::Regions`?
It will give you the same need to remember full path, this time - to constant.
So, what's the win? Code complication (need to keep that constant in some third file too, and remember about it's existence) and 15 chars economy?
**What that means** - my task loads in milliseconds. Task with the `task_name: :environment` will load a full app environment, which takes several seconds at least.